assets/js/blocks/product-gallery/inner-blocks/product-gallery-large-image/index.tsx

```diff
@@ -12,11 +11,9 @@ import { Edit, Save } from './edit';
 import metadata from './block.json';
 import './style.scss';
 
-if ( isExperimentalBuild() ) {
```

Comment:
The Product Gallery block contains some inner blocks. These inner blocks are only available inside the Product Gallery block, so even if I remove this, they will only be made available on the editor or frontend with its main block. 

The reason why I removed this is because of an issue that was occurring when using the `isExperimentalBuild()` (p1691416552602699/1691413062.084399-slack-C02UBB1EPEF) inside the inner blocks. I don't know exactly why this was happening, but I suspect the cause is that for the inner blocks to work, we have to import them in the `index.tsx` file of the [Product Gallery block](https://github.com/woocommerce/woocommerce-blocks/blob/chore/upgrade-webpack-to-version-5/assets/js/blocks/product-gallery/index.tsx#L14-L15), since there are many changes/improvements on the tree-shaken process in Webpack V5, sometimes Webpack can fail to recognize a file/piece of code that should be included in the bundle, in this particular case, it failed to include the `feature_flags` function that the `isExperimentalBuild()` call uses.

Reply:
> The Product Gallery block contains some inner blocks. These inner blocks are only available inside the Product Gallery block, so even if I remove this, they will only be made available on the editor or frontend with its main block.

Gotcha, thanks for clarifying.

> The reason why I removed this is because of an issue that was occurring when using the isExperimentalBuild() (p1691416552602699/1691413062.084399-slack-C02UBB1EPEF) inside the inner blocks. I don't know exactly why this was happening, but I suspect the cause is that for the inner blocks to work, we have to import them in the index.tsx file of the [Product Gallery block](https://github.com/woocommerce/woocommerce-blocks/blob/chore/upgrade-webpack-to-version-5/assets/js/blocks/product-gallery/index.tsx#L14-L15), since there are many changes/improvements on the tree-shaken process in Webpack V5, sometimes Webpack can fail to recognize a file/piece of code that should be included in the bundle, in this particular case, it failed to include the feature_flags function that the isExperimentalBuild() call uses.

👍 thanks for the detailed explanation! If this only affects inner blocks and won't cause any problems for the `product-gallery` or the `product-collection` blocks (these are the ones currently behind the experimental flag), all good.